frontend/src/components/NavLink.js

```diff
@@ -2,9 +2,11 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { NavLink as Link } from 'react-router-dom';
 
-function NavLink({ children, to, exact }) {
+function NavLink({
+  children, to, exact, fn,
+}) {
   return (
-    <Link className="usa-nav__link" to={to} activeClassName="usa-current" exact={exact}>
+    <Link className="usa-nav__link" to={to} activeClassName="usa-current" exact={exact} onClick={fn}>
```

Comment:
Just bringing this comment from #1017 over here so it doesn't get lost:

For the "/logout" path, we should either make a route that handles logging out when navigated to or switch this to be a button. We don't want to have links that act like buttons for accessibility reasons, and I was able to get split second 404's this way so it might also mess up or at least befuddle our analytics in that way